### PR TITLE
Update speaking-at-a-meetup.md

### LINKED ISSUE #5065: Update contact for dbt speakers and dbt blog the community section of the docs
### Contributions

- [X] I have read the contribution docs, and understand what's expected of me.

### Link to the page on docs.getdbt.com requiring updates

The [speaking at a meetup](https://docs.getdbt.com/community/resources/speaking-at-a-meetup) page might need some tweaking as the following items seem out of date or would benefit from some reformatting:
- Update the contacts for the blog post and speaker sections. (currently it refers to Janessa or Claire)
- Reformat the 'basic structure' sections as the headers look super small: https://docs.getdbt.com/community/resources/speaking-at-a-meetup#the-basic-structure

### What part(s) of the page would you like to see updated?

https://docs.getdbt.com/community/resources/speaking-at-a-meetup#responding-to-a-conference-call-for-speakers
https://docs.getdbt.com/community/resources/speaking-at-a-meetup#pair-it-with-a-blog-post

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by runleonarun**
closes https://github.com/dbt-labs/docs.getdbt.com/issues/5065

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding or removing pages (delete if not applicable):
- [ ] Add/remove page in `website/sidebars.js`
- [ ] Provide a unique filename for new pages
- [ ] Add an entry for deleted pages in `website/vercel.json`
- [ ] Run link testing locally with `npm run build` to update the links that point to deleted pages

### CODE REVIEW

File: website/docs/community/resources/speaking-at-a-meetup.md [hunk 1]
```diff
@@ -210,7 +208,7 @@ For virtual events: is there a poll you can launch, or a question you can throw
 
 ## Pair it with a blog post
 
-The hardest part of nailing a great talk is the content, so if you’ve made it this far, you’ve already done most of the work. Turning your content into a blog post is a great way to solidify your thinking, and get some extra exposure. If you’d like to be features on the [dbt Blog](https://blog.getdbt.com/), reach out to us (@Claire and @Janessa) on Slack.
+The hardest part of nailing a great talk is the content, so if you’ve made it this far, you’ve already done most of the work. Turning your content into a blog post is a great way to solidify your thinking, and get some extra exposure. If you’d like to be featured on the [dbt Blog](https://blog.getdbt.com/), reach out to us on Slack.
```
Comment (runleonarun) [inline website/docs/community/resources/speaking-at-a-meetup.md]:
```suggestion
The hardest part of nailing a great talk is the content, so if you’ve made it this far, you’ve already done most of the work. Turning your content into a blog post is a great way to solidify your thinking, and get some extra exposure. If you’d like to be featured on the [dbt Blog](https://blog.getdbt.com/), please email us at [community@dbtlabs.com](mailto:community@dbtlabs.com).
```

File: website/docs/community/resources/speaking-at-a-meetup.md [hunk 2]
```diff
@@ -246,4 +244,4 @@ Do any audience members use a communication device? Can you share your slides ah
 
 ### Responding to a conference Call for Speakers
 
-If you’re submitting a response for a Call for Speakers, and talking about dbt, we’re happy to work with you on this. Reach out to us (@Claire and @Janessa) in Slack to let us know!
+If you’re submitting a response for a Call for Speakers, and talking about dbt, we’re happy to work with you on this. Reach out to us in Slack to let us know!
```
Comment (runleonarun) [inline website/docs/community/resources/speaking-at-a-meetup.md]:
```suggestion
If you’re submitting a response for a Call for Speakers, and talking about dbt, we’re happy to work with you on this. You can email us at [community@dbtlabs.com](mailto:community@dbtlabs.com) for more information.
```

File: website/docs/community/resources/speaking-at-a-meetup.md [hunk 3]
```diff
@@ -246,4 +244,4 @@ Do any audience members use a communication device? Can you share your slides ah
 
 ### Responding to a conference Call for Speakers
 
-If you’re submitting a response for a Call for Speakers, and talking about dbt, we’re happy to work with you on this. Reach out to us (@Claire and @Janessa) in Slack to let us know!
+If you’re submitting a response for a Call for Speakers, and talking about dbt, we’re happy to work with you on this. You can email us at [community@dbtlabs.com](mailto:community@dbtlabs.com) for more information.
```
Comment (nghi-ly) [inline website/docs/community/resources/speaking-at-a-meetup.md]: sugg: may (vs can) to express permission (vs ability) 

```suggestion
If you’re submitting a response for a Call for Speakers, and talking about dbt, we’re happy to work with you on this. You may email us at [community@dbtlabs.com](mailto:community@dbtlabs.com) for more information.
```

**Review by nghi-ly — APPROVED** [review]
looks great! ❤️ ❤️ ❤️ the rearchitecture of the section headings. left one non-blocking sugg

**Pull request merged**